src/onefetch/repo.rs

```diff
@@ -63,7 +63,7 @@ impl<'a> Repo<'a> {
         number_of_commits.to_string()
     }
 
-    pub fn get_authors(&self, n: usize) -> Vec<(String, usize, usize)> {
+    pub fn get_authors(&self, n: usize, include_email: bool) -> Vec<(String, Option<String>, usize, usize)> {
```

Comment:
Looks like the format check still failed. I think it's this line being too long. Try listing the function arguments in multiple lines.